LayoutTests/http/tests/referrer-policy/resources/document.html

```diff
@@ -2,7 +2,13 @@
 <html>
 <body>
 <script>
+// If loaded in cross origin frame with site isolation, then this will run in another WCP
+// and need to have the tracking prevention adjusted as well.
```

Comment:
We're hoping to move away from running all tests in a cross-origin iframe with site isolation on.  Could you take this case and add it as a new test that loads in a cross-origin iframe instead of modifying this test?